package.json

```diff
@@ -69,10 +69,10 @@
     "eslint-config-airbnb": "1.0.2",
     "eslint-plugin-react": "^4.1.0",
     "mocha": "^2.2.5",
-    "redux": "^3.4.0",
+    "redux": "~4.0.0",
```

Comment:
Don't use `~` here, it'd prevent updating to new minor versions of redux. Use `^`.